runExample.sh

```diff
@@ -2,7 +2,7 @@
 
 # Usage:
 # ./runExample -t transformExample
-# ./runExample -u toUrlExample
+# ./runExample -c connectionExample
```

Comment:
I missed this getting into the project when you first added it. I agree that this script is very useful, but I'd prefer to see it written using node.js.

Not a deal-breaker, just a strong preference ;)

Reply:
Makes sense. It was originally just a consolidation of the shell commands I was already using to test the examples, and it was just for my own use. I'll create a story to convert it.